### PR TITLE
Rm11bis correction on barbarismes courants

### PR DESCRIPTION
**Pull request by remymartinDev**

### CODE REVIEW

**Review by udomai — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -50272,12 +50272,12 @@ though, and has since been slightly modified)-->
                 <example correction="leurs vélos">C'est <marker>les vélos à eux</marker>.</example>
             </rule>
         </rulegroup>
-        <rulegroup id="BARBARISMES_COURANTS" name="Barbarismes courants divers" default="temp_off">
+        <rulegroup id="BARBARISMES_COURANTS" name= "Barbarismes courants divers" default="temp_off">
```
[the remark this answers is not in the document]
Comment (remymartinDev) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml]: this has been done by accident, and not on purpose

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -50272,12 +50272,12 @@ though, and has since been slightly modified)-->
...
             <rule>
                 <pattern>
                     <token regexp="yes">antidiluvien(nes?|s)?</token>
                 </pattern>
-                <message>Une erreur d'orthographe semble être détectée.</message>
+                <message>Une erreur d’orthographe semble être détectée.</message>
```
[the remark this answers is not in the document]
Comment (remymartinDev) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml]: this has been done by accident, and not on purpose

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 3]
```diff
@@ -50286,9 +50286,9 @@ though, and has since been slightly modified)-->
                 <pattern>
                     <token regexp="yes">arborigènes?</token>
                 </pattern>
-                <message>Une erreur d'orthographe semble être détectée.</message>
+                <message>Une erreur d’orthographe semble être détectée.</message>
                 <suggestion><match no="1" regexp_match="(a)r(borigènes?)" regexp_replace="$1$2"/></suggestion>
-                <example correction="aborigène">Nous avons rencontré une tribu <marker>arborigène</marker>.</example>
+                    <example correction="aborigène">Nous avons rencontré une tribu <marker>arborigène</marker>.</example>
```
[the remark this answers is not in the document]
Comment (remymartinDev) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml]: this has been done by accident, and not on purpose

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 4]
```diff
@@ -50663,11 +50663,11 @@ though, and has since been slightly modified)-->
...
             <rule>
                 <pattern>
-                    <token>opprobe</token>
+                    <token regexp="yes">opprobes?</token>
                 </pattern>
                 <message>Une erreur d'orthographe semble être détectée.</message>
-                <suggestion>opprobre</suggestion>
-                <example correction="opprobre">L'attitude inqualifiable du député jette l'<marker>opprobe</marker> sur toute la classe politique.</example>
+                <suggestion><match no="1" regexp_match="(opprob)(es?)" regexp_replace="$1r$2"/></suggestion>
+                <example correction="opprobre">L’attitude inqualifiable du député jette l'<marker>opprobe</marker> sur toute la classe politique.</example>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:50670]: no need for fancy apostrophes – regular apostrophes are transformed automatically in the message

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 5]
```diff
@@ -50633,12 +50633,12 @@ though, and has since been slightly modified)-->
...
                     <marker>
-                        <token regexp="yes">ast[eé]rix</token>
+                        <token regexp="yes" >ast[eé]rix</token>
                     </marker>
                 </pattern>
-                <message>Une erreur d'orthographe semble être détectée.</message>
+                <message>Une erreur d’orthographe semble être détectée.</message>
                 <suggestion>astérisque</suggestion>
-                <example correction="astérisque">Le mot est marqué d'une <marker>astérix</marker>.</example>
+                    <example correction="astérisque">Le mot est marqué d'une <marker>astérix</marker>.</example>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:50641]: wrong indent

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 6]
```diff
@@ -50633,12 +50633,12 @@ though, and has since been slightly modified)-->
                 <pattern>
                     <token regexp="yes">des|une|l'</token>
                     <marker>
-                        <token regexp="yes">ast[eé]rix</token>
+                        <token regexp="yes" >ast[eé]rix</token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:50636]: this space is not necessary

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 4 above)
Comment (remymartinDev) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:50670]: I'm experiencing difficulties to understand how those different style apostrophes are appearing ...

**Review by udomai — APPROVED** [review]

**Pull request merged**